### ISSUE TITLE
Archive Failure. Possibly due to un-escaped white space.

### ISSUE THREAD

**Issue by JettF** [CocoaPods/CocoaPods#6228]
🌈

> Before you start, are you using the latest CocoaPods release?
> A lot changes with Xcode releases that are not backwards compatible.
>
> Xcode 8 Status: Only works with CocoaPods 1.1.0+
> Pushing a Swift 2.3 Pod: You need to add a `.swift-version` file or the argument `--swift-version=VERSION`, see the CHANGELOG
> Not an issue about the CocoaPods command line app? Please file an issue in the appropriate repo - https://github.com/CocoaPods
> Issues are for feature requests, and bugs; questions should go to Stack Overflow
> Using CocoaPods <= 0.39: http://blog.cocoapods.org/Sharding/

* [ ] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

Attempted to archive my application. 

## What did you expect to happen?

The archive to complete successfully.

## What happened instead?

Archiving failed.

The target and scheme of my application are both named "My App". I believe that the CocoaPods shell script to embed the pods is not properly escaping the space, which caused the archive to fail. At the bottom of the error message pasted below you can see that the script was looking for "My" instead of "My_App".

Here is the error message: 
```
PhaseScriptExecution [CP]\ Embed\ Pods\ Frameworks /Users/JettF/Library/Developer/Xcode/DerivedData/My_App-gragjdwfnyzchsajwcvrwhxsjwls/Build/Intermediates/ArchiveIntermediates/My_App\ Development/IntermediateBuildFilesPath/My_App.build/Release-iphoneos/My_App\ Development.build/Script-DB68A023C7EDA602298A50EC.sh
    cd /Users/JettF/Documents/Repos/My_App
    /bin/sh -c \"/Users/JettF/Library/Developer/Xcode/DerivedData/My_App-gragjdwfnyzchsajwcvrwhxsjwls/Build/Intermediates/ArchiveIntermediates/My_App\ Development/IntermediateBuildFilesPath/My_App.build/Release-iphoneos/My_App\ Development.build/Script-DB68A023C7EDA602298A50EC.sh\"

mkdir -p /Users/JettF/Library/Developer/Xcode/DerivedData/My_App-gragjdwfnyzchsajwcvrwhxsjwls/Build/Intermediates/ArchiveIntermediates/My_App Development/BuildProductsPath/Release-iphoneos/My_App.app/Frameworks
Symlinked...
rsync -av --filter "- CVS/" --filter "- .svn/" --filter "- .git/" --filter "- .hg/" --filter "- Headers" --filter "- PrivateHeaders" --filter "- Modules" "/Users/JettF/Library/Developer/Xcode/DerivedData/My_App-gragjdwfnyzchsajwcvrwhxsjwls/Build/Intermediates/ArchiveIntermediates/My_App Development/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/AMEdgeView.framework" "/Users/JettF/Library/Developer/Xcode/DerivedData/My_App-gragjdwfnyzchsajwcvrwhxsjwls/Build/Intermediates/ArchiveIntermediates/My_App Development/InstallationBuildProductsLocation/Applications/My_App.app/Frameworks"
building file list ... done
AMEdgeView.framework/
AMEdgeView.framework/AMEdgeView
AMEdgeView.framework/Info.plist

sent 528492 bytes  received 70 bytes  1057124.00 bytes/sec
total size is 528186  speedup is 1.00
Code Signing /Users/JettF/Library/Developer/Xcode/DerivedData/My_App-gragjdwfnyzchsajwcvrwhxsjwls/Build/Intermediates/ArchiveIntermediates/My_App Development/InstallationBuildProductsLocation/Applications/My_App.app/Frameworks/AMEdgeView.framework with Identity iPhone Distribution: MyCompany, LLC
/usr/bin/codesign --force --sign   --preserve-metadata=identifier,entitlements /Users/JettF/Library/Developer/Xcode/DerivedData/My_App-gragjdwfnyzchsajwcvrwhxsjwls/Build/Intermediates/ArchiveIntermediates/My_App Development/InstallationBuildProductsLocation/Applications/My_App.app/Frameworks/AMEdgeView.framework
/Users/JettF/Library/Developer/Xcode/DerivedData/My_App-gragjdwfnyzchsajwcvrwhxsjwls/Build/Intermediates/ArchiveIntermediates/My: No such file or directory
```

### Stack

```
   CocoaPods : 1.2.0.beta.1
        Ruby : ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
    RubyGems : 2.5.1
        Host : Mac OS X 10.12.1 (16B2555)
       Xcode : 8.1 (8B62)
         Git : git version 2.9.3 (Apple Git-75)
Ruby lib dir : /usr/local/Cellar/ruby/2.3.1/lib
`private Repos`
Repositories : master - https://github.com/CocoaPods/Specs.git @ 75c4751f4bccafb8c9e3381b496ad14cb77b958b
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.1
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.1.1
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
platform :ios, '9.0'

use_frameworks!
inhibit_all_warnings!

source 'Private Repo'
source 'https://github.com/CocoaPods/Specs.git'

def fabric_pods
    pod 'Fabric'
    pod 'Crashlytics'
end

def import_pods    
    pod 'ActionSheetPicker-3.0', '~> 2.2'
    pod 'AlamofireNetworkActivityIndicator', '~> 2.0'
    pod 'AMOrientationUpdatingImageView', '~> 1.1'
    pod 'AMResizingTextView', '~> 1.0'
    pod 'AMTextFieldPickerExtension', '~> 0.6'
    pod 'AmazonS3RequestManager', '~> 1.0'
    pod 'APNumberPad', :git => 'https://github.com/JettF/APNumberPad.git', :branch => 'master'
    pod 'AssociatedValues', '~> 2.0'
    pod 'Branch', '~> 0.12'
    pod 'BRYXBanner', '~> 0.7'
    pod 'CrashlyticsRecorder', '~> 2.0'
    pod 'DKImagePickerController', '~> 3.4'
    pod 'FDTake', '~>0.4'
    pod 'GooglePlaces'
    pod 'Harpy', '~> 3.3'
    pod 'MBProgressHUD', '~> 1.0'
    pod 'Obfuscator', :git => 'https://github.com/pjebs/Obfuscator-iOS.git', :branch => 'master'
    pod 'RateLimit', '~> 1.3'
    pod 'ReactiveKit', '~> 3.0'
    pod 'Bond', '~> 5.0'
    pod 'StatefulViewController', '~> 3.0'
    pod 'STPopup', '~> 1.8'
    pod 'Willow', '~> 2.0'
    pod 'XLPagerTabStrip-AnthonyMDev', '~> 2.0'
  
end

target :'My App Release' do
    fabric_pods
    import_pods
end

target :'My App Development' do
    fabric_pods
    import_pods
end

abstract_target :'Test' do
    import_pods
    
    pod 'Nocilla'
    pod 'Nimble'
   
    target :'My AppTests' do
        pod 'AMTestHelpers', '~> 1.0'
    end
    
    target :'My AppPerformanceTests' do
        
    end
    
end

post_install do |installer|
    installer.pods_project.targets.each do |target|
        if target.name == 'RateLimit' ||
            target.name == 'AlamofireImage' ||
            target.build_configurations.each do |config|
                config.build_settings['ENABLE_TESTABILITY'] = 'YES'
            end
        end
    end
end
```

**Comment by benasher44** [comment]
Hi there! This issue was fixed in #6160. The fix is on master, and it will be available in an upcoming release. Thanks for filing!

**Comment by JettF** [comment]
Awesome. Thank you!
